### ISSUE TITLE
Tell user if login failed

### ISSUE THREAD

**Issue by jccaswell** [TPM-Repos/ModifiedCorporateTheme#13]
Currently on the login screen it simply displays the "There has been an issue" if the user types in an invalid username or password. This is the same error message for several other issues too.

**Comment by joelhanneman** [comment]
Is there an upvote mechanism in Issues?  If so, I would upvote this one.  By now, I know what this message could mean, but for folks who don't this will be pretty frustrating..

**Comment by joelhanneman** [comment]
Is this the solution, implemented in Templates site?  Def an improvement!  In this case I entered bogus creds.

Is there a different message if there's a different issue?

![image](https://github.com/user-attachments/assets/eb0d9517-ea66-478d-a240-cc25e6bb7c2e)

**Comment by jccaswell** [comment]
The text that shows there is easily configurable: we can set it to whatever (as long as it fits)

I have closed this issue due to other improvement I made previously:
The login button is not available until a connection is made to the server. If no connection can be made then login is not allowed and a different message appears. 

For security reasons the API does not report whether it is the username or the password that is incorrect. Even if they are correct, if the user is "disabled" the API will return the same message.
